src/sentry/static/sentry/app/components/dashboards/widgetQueryFields.tsx

```diff
@@ -115,16 +115,16 @@ function WidgetQueryFields({displayType, errors, fields, fieldOptions, onChange}
               borderless
               onClick={event => handleRemove(event, i)}
               icon={<IconDelete />}
-              title={t('Remove this overlay')}
-              label={t('Remove this overlay')}
+              title={t('Remove this Y-Axis')}
+              label={t('Remove this Y-Axis')}
```

Comment:
I think renaming overlay to Y-axis makes sense here. We use y-axis across performance, and discover so I hope that users will have some familiarity with it here too.

Reply:
I'll be adding some information to the product docs about this.